presto-docs/src/main/sphinx/aws/emr.rst

```diff
@@ -0,0 +1,132 @@
+
+4. In order to take advantage of the extra resources of the new node, you
+may need to edit some of the Presto memory settings (for example
+``query.max.memory``). For more information on configuring Presto using
+presto-admin please refer to the Configuration section of the presto-admin
+:doc:`/installation/presto-admin/quick-start-guide`.
+
+5. If you made configuration changes, make sure to deploy those changes to
+all nodes:
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem configuration deploy
+
+6. If you did not make any configuration changes in step 4 then just start
+Presto on the new node:
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem server start -H new_node_fqdn
+
+If you did make configuration changes in step 4 then all nodes need to be restarted
+(note that you don't need to also start the new node because it is now part
+of the topology and it will be started as part of the restart):
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem server restart
+
+7. To verify that the new node was correctly started run:
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem server status
+
+Removing a node
+---------------
+
+1. Stop Presto on the worker node you would like to remove:
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem server stop -H node_to_remove_fqdn
+
+2. Uninstall Presto from the worker node:
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem server uninstall -H node_to_remove_fqdn
+
+3. Remove the FQDN of the node you would like to remove from the presto-admin
+topology file. By default this file is located at ``/etc/opt/prestoadmin/
+config.json``.
+
+4. To adjust your Presto installation to the decrease in resources you
+may need to edit some of the Presto memory settings (for example
+``query.max.memory``). For more information on configuring Presto using
+presto-admin please refer to the Configuration section of the presto-admin
+:doc:`/installation/presto-admin/quick-start-guide`.
+
+5. If you made configuration changes, make sure to deploy those changes to
+all nodes and then restarting them by running:
+
+.. code-block:: bash
+
+    sudo /path/to/presto-admin -i /path/to/your.pem configuration deploy
+    sudo /path/to/presto-admin -i /path/to/your.pem server restart
+
+6. Terminate the node using the AWS console or the AWS CLI tool.
+
+Troubleshooting
+===============
+
+* If you are using the RPM for installing Presto, make sure you use the RPM 
+built for RPM Package Manager version > 4.6. If you are using an RPM which 
+is not compatible with an EMR node, you may get the following error during 
+installation:
+
+.. code-block:: none
+
+    headerRead failed: hdr load: BAD
+   
+* EMR deploys ``hive 1.0.0`` version which has a bug with 
+``ALTER TABLE..RENAME TO`` command. Thus while using Presto's Hive Connector 
+you get:
+
```

Comment:
I think the following text will be a bit more clarifying:

EMR deploys ``hive 1.0.0`` version which has a bug with ``ALTER TABLE..RENAME TO`` command. When running this command in Presto using the Hive connector, you may run into the following: